src/cloud/project/project-patch.md

```diff
@@ -35,37 +29,165 @@ All patch file names must end with the `.patch` extension.
+║                │                                                 │          │             │  - magento/framework            ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ BUNDLE-2554    │ Set Payment info bug                            │ Required │ Not applied │ Affected components:            ║
+║                │                                                 │          │             │  - amzn/amazon-pay-module       ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-1           │ Fixes issue 1                                   │ Optional │ Applied     │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-2           │ Fixes issue 2                                   │ Optional │ Not applied │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-3           │ Fixes issue 3                                   │ Optional │ Not applied │ Required patches:               ║
+║                │                                                 │          │             │  - MC-2                         ║
+║                │                                                 │          │             │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ N/A            │ ../m2-hotfixes/MDVA_custom__2.3.5_ce.patch      │ Custom   │ N/A         │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-framework     ║
+╚════════════════╧═════════════════════════════════════════════════╧══════════╧═════════════╧═════════════════════════════════╝
+Magento 2 Enterprise Edition, version 2.3.5.0
+```
+
+The status table contains the following types of information:
+
+-  **Type**:
+
+   -  `Optional`—All individual patches from the MQP package and the [Magento Cloud Patches]({{ site.baseurl }}/cloud/project/project-patch.html) package are optional for {{ site.data.var.ee }} and {{ site.data.var.ce }} installations.
+   -  Required—All individual patches from the {{ site.data.var.mcp-prod }} package are required for Cloud customers.
```

Comment:
Ha! I only made a suggestion on the first one, but if you liked that suggestion then you probably need to update all the others in the list. 😊

```suggestion
   -  `Required`—All individual patches from the {{ site.data.var.mcp-prod }} package are required for Cloud customers.
```